src/components/dev-hub/badge.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import Badge from './badge';
+
+<Meta title="Badge" />
+
+# Badges
```

Comment:
Not sure if we are using these anymore? We might just want to remove the `Badge` entirely.

Reply:
Oh yea didn't realize we weren't using it anywhere lol Its actually not even in the design anymore 🤔 . I'll go ahead and remove it.